docs/UserGuide.md

```diff
@@ -159,18 +159,19 @@ Examples:
 
 Assign the specified patient into the specified appointment date and time.
 
-Format: `assign INDEX d/DATE t/TIME`
+Format: `assign INDEX d/DATE t/TIME [dur/DURATION]`
 
 * Puts the patient at the specified INDEX into an appointment time slot.
 * The INDEX refers to the index number indicated in the patient list.
 * The INDEX **must be a positive integer** 1, 2, 3, …​
 * The `DATE` and `TIME` of the appointment must be included.
-* The timeslot indicated by `DATE` and `TIME` must be available.
+* The `DURATION` is in unit of minute, with a default value of `60` when the term is omitted.
```

Comment:
The phrasing seems a bit off.  
Would it be better if we phrase it as `The 'DURATION' is measured in minutes and will be defaulted to 60 minutes if omitted`?

Do add on if the phrasing can be improved further. 👍

Reply:
Jin Hao's phrasing looks good to me too 👍🏻

Reply:
Yes that is good indeed, I have changed it.